azerothcore/azerothcore-wotlk
# fix(DB/creature_loot): Defias Rapier drops

## Changes Proposed:
- Removes the [Defias Rapier](https://tbc.wowhead.com/item=1925/defias-rapier) from RLT 24077, where it was being incorrectly shared amongst 462 NPCs. It should only be dropped by [Defias Watchman](https://tbc.wowhead.com/npc=1725/defias-watchman).
- Added the Rapier directly to the Watchman's drop table with a drop chance of 6.6% as a member of loot group 1.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/8818
- Closes https://github.com/chromiecraft/chromiecraft/issues/2213

## SOURCE:

https://tbc.wowhead.com/item=1925/defias-rapier#dropped-by

Wowwiki [Defias Rapier](https://wowwiki-archive.fandom.com/wiki/Defias_Rapier?oldid=2027400)
> This item drops from Defias Watchman in The Deadmines.

## Tests Performed:

- Ran SQl, no errors/warnings, correct number of rows changed/deleted.

## How to Test the Changes:


```SQL
SELECT rlt.entry, it.name
FROM `reference_loot_template` rlt
JOIN `item_template` it ON rlt.Item = it.entry
WHERE rlt.item = 1925;
```
Should now return an empty set.

```SQL
SELECT ct.name, clt.chance, ct.maxlevel, it.ItemLevel
FROM `creature_template` ct
JOIN `creature_loot_template` clt ON ct.lootid = clt.entry
JOIN `item_template` it ON clt.item = it.entry
WHERE it.entry = 1925;
```
Should now return Defias Watchman with 6.6% drop rate.

## Known Issues and TODO List:


N/A.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
